testjson/execution.go

```diff
@@ -471,6 +472,17 @@ func (e *Execution) Elapsed() time.Duration {
 	return clock.Now().Sub(e.started)
 }
 
+func (e *Execution) ElapsedOnWrite() time.Duration {
```

Comment:
What's the purpose of this? Does `Elapsed()` not give the correct value for some reason?

Reply:
I'm still curious about this. I think we can find a better way to solve this than introducing two new methods on `Execution`.